core/arch/arm32/kernel/tee_ta_manager.c

```diff
@@ -625,6 +625,9 @@ static TEE_Result tee_ta_load(const kta_signed_header_t *signed_ta,
 #if defined(CFG_SE_API)
 	ctx->se_service = NULL;
 #endif
+#if !defined(CFG_TEE_CORE_NO_TA_TRACE)
+	ctx->trace = true;
+#endif
```

Comment:
Yes. But this is for future usage.

I'll rework this patch to remove the "trace" bool in the context. We'll not be able to enable/disable it at runtime.

I'll also change CFG_TEE_CORE_NO_TA_TRACE into CFG_TEE_CORE_TA_TRACE to avoid double negation.